libparsec/crates/client/src/workspace/store/block_access.rs

```diff
@@ -0,0 +1 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
```

Comment:
This file is mostly empty

Reply:
good catch ! this is a dummy file (not even including in the mod.rs)